src/reynir_correct/pattern.py

```diff
@@ -188,6 +221,28 @@ def get_wordform(word: str, lemma: str, cat: str, variants: Iterable[str]) -> st
         # Can be many possible word forms; we want the first one in most cases
         return wordforms[0].bmynd
 
+    def get_wordform(self, lemma, cat, variants):
+        """Get correct wordform from BinPackage,
+        given a set of variants"""
+
+        # Get rid of argument variants in verbs:
+        variants = list([x for x in variants if not x.isdigit()])
```

Comment:
Either list() or [] are superfluous, only one of the two is required

Reply:
...in fact the isdigit() filter is done twice. This could be:

`realvars = [x for x in variants if not x.isdigit() and x not in SKIPVARS]`

entirely dropping the for loop.

Reply:
Thank you, done!